lime/js/predicted_value.js

```diff
@@ -0,0 +1,138 @@
+import d3 from 'd3';
```

Comment:
The bar in this visualization seems to be shifted [way to the right](https://lh4.googleusercontent.com/nnFCZUGg_FX3gKiVL3SAkAQVoPpQCOGPG9U1_jbwyn_DopNkwfY8Gc-kuxpmUROp6EKP4BTIj5_XXyM=w2880-h1464), it seems that it should be under 'Predicted value'